tensorflow_serving/model_servers/http_server.cc

```diff
@@ -180,6 +181,9 @@ class RestApiRequestDispatcher {
 std::unique_ptr<net_http::HTTPServerInterface> CreateAndStartHttpServer(
     int port, int num_threads, int timeout_in_ms,
     const MonitoringConfig& monitoring_config, ServerCore* core) {
+  // Ignore SIGPIPE and allow errors to propagate through error codes.
```

Comment:
i think this is specific to libevent. it will be good to move this closer to libevent initialization here:

https://github.com/tensorflow/serving/blob/9578f3d10c786c6714b9a8b481dd74f454402477/tensorflow_serving/util/net_http/server/internal/evhttp_server.cc#L46

Reply:
ok